ArjunaJTA/jta/classes/com/arjuna/ats/jta/common/JTAEnvironmentBean.java

```diff
@@ -113,7 +115,11 @@
 
 	private Map<String, Integer> commitMarkableResourceRecordDeleteBatchSizeMap = new HashMap<String, Integer>();
 
-	private static final String defaultTransactionOperationsProviderClassName = ServerVMClientUserTransactionOperationsProvider.class.getName(); 
+	private static final String defaultSubordinateTransactionImporterClassName = NarayanaSubordinateTransactionImporter.class.getName();
```

Comment:
they relates indirectly. They use the same way how params are defined in the `JTAEnvironmentBean` but they are two different settings

`defaultTransactionOperationsProviderClassName` and `defaultSubordinateTransactionImporterClassName`

just I'm not sure if I respond to your question or not.